source/administration/config-settings.rst

```diff
@@ -1453,7 +1453,7 @@ Allow Users to View Archived Channels (Beta)
 **False**: Users are unable to view, share or search for content of channels that have been archived.
 
 +-------------------------------------------------------------------------------------------------------------------------------------------------------+
-| This feature's ``config.json`` setting is ``"ViewArchivedChannels": false`` with options ``true`` and ``false``.                                      |
+| This feature's ``config.json`` setting is ``"ExperimentalViewArchivedChannels": false`` with options ``true`` and ``false``.                                      |
```

Comment:
The table is not rendering anymore. You can fix the table formatting by changing the "Line Wrap Mode" to "No Wrap" and then editing the table so that it is aligned.

Reply:
Ah...sorry for my fix without any validations. I've fixed the table formatting by [this commit](https://github.com/mattermost/docs/pull/3278/commits/0a909a4d9f244ae44ab2df46d3f7afa0afd0d3bf) and confirmed that the table is rendering by building html.  Thanks!